dock_container.go

```diff
@@ -50,6 +50,9 @@ func NewDockContainer(dock *Dock, dockable Dockable) *DockContainer {
 	d.AddChild(d.content)
 	return d
 }
+func (d *DockContainer) Header() *dockHeader {
```

Comment:
Returning a private struct from a public API is not acceptable. Why do you feel you need to get access to this private, internal, field? It is being protected specifically to help prevent unexpected changes to its state and as such, I'm unlikely to want it exposed.

Reply:
Without this action there is no way to fetch the dock title's panel